src/main/java/org/magicdgs/readtools/cmd/RTStandardArguments.java

```diff
@@ -37,6 +35,53 @@
     /** Cannot be instantiated. */
     private RTStandardArguments() {}
 
+    //////////////////////////////////
+    // OLD-STYLE GATK ARGUMENTS
+    // TODO - this arguments should be substituted by the GATK defaults for addressing
+    // TODO - https://github.com/magicDGS/ReadTools/issues/346
```

Comment:
Change for https://github.com/magicDGS/ReadTools/issues/364 and https://github.com/magicDGS/ReadTools/issues/340